projects/opendr_ws/src/opendr_perception/scripts/panoptic_segmentation_efficient_lps.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
```

Comment:
This is required to run smoothly on embedded devices.
```suggestion
#!/usr/bin/env python3
```